proxy/proxy.py

```diff
@@ -184,6 +184,9 @@ def initialize(input_args: Optional[List[str]]
             sys.exit(0)
 
         # Setup logging module
+        args.log_file  = cast(Optional[str], opts.get('log_file', args.log_file))
+        args.log_level  = cast(Optional[str], opts.get('log_level', args.log_level))
+        args.log_format  = cast(Optional[str], opts.get('log_format', args.log_format))
```

Comment:
These flags already exists.  See https://github.com/abhinavsingh/proxy.py#flags

Can you expand why exactly are you trying to add them here again.

Thank you!!!

PS: Apologies for delay, been very tied up with other projects.

Reply:
Yes, sure. Ok, these flags work perfect when supplied from command line. For example, this works perfect: 

```
proxy --log-file /tmp/filer.txt ....
```

However, if you do the same in Python:
```python
import proxy
proxy.main(
    hostname=...,
    port=8080,
    log_file="/tmp/filer.txt"
)
```

It doesn't work. The above statement fixes that issue.

Reply:
However, i just realized now that linux & mac doesn't startupinfo parameter for subprocess module. I guess, we will have to modify the pull request.